Library/Homebrew/test/Gemfile

```diff
@@ -1,9 +1,15 @@
 source "https://rubygems.org"
 
-gem "mocha", "~> 1.1"
-gem "minitest", "~> 5.3"
-gem "rake", "~> 10.3"
-gem "parallel_tests", "~> 2.9"
+group :test do
+  gem "minitest", "~> 5.9"
+  gem "minitest-reporters"
+  gem "mocha", "~> 1.2"
+  gem "rake", "~> 10.5"
+  gem "rspec", "~> 3.5.0"
```

Comment:
I'm very much in favor of the RSpec syntax. Still, we will need to get both working first in order to transition from one to the other. And installing everything in one place is certainly better than it was before, where everything was duplicated also in `cask/vendor/bundle`.